loader/language.cpp

```diff
@@ -95,14 +95,14 @@ const MultiLang lang_fav_added[] = {
 	{LANG_EN, "Added to favorites"},
 	{LANG_DE, "Favorit hinzugef�gt"},
 	{LANG_FR, "Ajout� aux favoris"},
-	{LANG_ES, "Favorito agregado"},
+	{LANG_ES, "Agregado a favoritos"},
 	{LANG_NL, "Toegevoegd aan favs"},
 };
 
 const MultiLang lang_fav_removed[] = {
 	{LANG_EN, "Removed favorite"},
 	{LANG_DE, "Favorit entfernt"},
 	{LANG_FR, "Retir� des favoris"},
-	{LANG_ES, "Favorito eliminado"},
+	{LANG_ES, "Eliminado de favoritos"},
```

Comment:
and this is three characters too long

Reply:
ok!, no problem , I want to clarify , the weren't wrong, just in spanish there is a lot of ways to say the same , so we can keep them as were before